### ISSUE TITLE
NVDA should have an easy way to put it on a u3drive

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#197]
**Reported by mike.reiser on 2008-10-15 16:00**
Currently, inorder to allow NVDA to operate successfully on a u3 drive, one currently needs to create an autorun.inf file by hand which points to the nvda.exe file on the u3 drive.  While this autorun file is easy to create for experienced computer users, it seems to me that a beginner or somewhat experienced user would be un comfortable doing this task and should have an intuitive way of doing this.  I can invision either a utility built into NVDA or packaged with it that would do the following.  
1.  Ask the user for the location of there u3 drive.  
2.  When the u3 location is given, copy the nvda files and create an autorun.inf file for the user that will launch nvda when the drive is insirted.  System access has something like this for copying it to a u3 drive and this would be great for nvda.  Of course it should throw an error if a non u3 drive is attempted.  I'm not sure if this should be packaged with the official nvda or the portable version.  If there is a program for doing this already feel free to put it here, I just thought I'd open this for discussion.

**Comment by derekriemer** [comment]
Is this still relevant since modern computers won't execute programs from usb's even if asked to do so, without user intervention?

**Comment by leonardder** [comment]
I've read on the Dutch Wikipedia that U3 has been obsolete for many years now. I'd like to vote for a won't fix
